JitsiConference.js

```diff
@@ -2689,29 +2692,6 @@ JitsiConference.prototype.setReceiverVideoConstraint = function(
     this.rtc.setReceiverVideoConstraint(maxFrameHeight);
 };
 
-/**
- * Get video SIP GW handler, if missing will create one.
- *
- * @returns {VideoSIPGW} video SIP GW handler.
- */
-JitsiConference.prototype._getVideoSIPGWHandle = function() {
-    if (!this.videoSIPGWHandler) {
-        this.videoSIPGWHandler = new VideoSIPGW(this.room);
-        logger.info('Created VideoSIPGW');
-    }
-
-    return this.videoSIPGWHandler;
-};
-
-/**
- * Checks whether video SIP GW service is available.
- *
- * @returns {boolean} whether video SIP GW service is available.
- */
-JitsiConference.prototype.isVideoSIPGWAvailable = function() {
```

Comment:
You got rid of isVideoSIPGWAvailable because there were no known uses in the jitsi-meet app?

Reply:
Yes